cli/volume.go

```diff
@@ -157,7 +157,7 @@ func (v *VolumeRemoveCommand) Init(c *Cli) {
 	v.cli = c
 
 	v.cmd = &cobra.Command{
-		Use:     "remove [volume]",
+		Use:     "remove <volume name>",
 		Aliases: []string{"rm"},
 		Short:   "Remove pouch volumes",
 		Args:    cobra.MinimumNArgs(1),
```

Comment:
I think we need to more careful with the command args. And consistent with `Use` and `Args`.
You said `Remove pouch volumes`, does this support removing multi volumes?
If that the args `cobra.MinimumNArgs(1)` is correct, otherwise inconsistency happens.

Reply:
We should support removing multi volumes, just now not support.

Reply:
Updated, PTAL @allencloud